### DISCUSSION TITLE
Two Different Mode for Downloading and Total Downloading Size

### DISCUSSION THREAD

**Discussion by NotSoMrAnderson** [Dineshkarthik/telegram_media_downloader#143] [Ideas]
is it possible to add interval id download and certain ids to download and at the final step see a confirmation message with total size of downloading files..

**Comment by NotSoMrAnderson** [comment]
any thoughts ?

**Comment by Dineshkarthik** [comment]
Hi,

You already can use the interval id like set the message id from which you want to start downloading as the `last_read_message_id` in config.yaml. For specific ids alone to download I am hesitant as this complicates the simple download script (I know this is just a additional field in the configuration yaml, but if you check the closed issues, discussions, telegram support channel how many users find it hard to use this current setup) but I will definitely look into this.

For the download size are you looking for a live progress bar kind of setup or a log at the end of download stating the total downloaded size?

**Comment by NotSoMrAnderson** [comment]
about the first one, it's not an actual interval because we can't choose until which message-id script should continue downloading and it will download all of the files from last_read_message_id until the end of the chat, About the second one hope you find a way to add without make script "hard to use", and the third one, neither of those kinda confirming console messages that indicates the total size of files that script going to download and take a yes or no (either simple typing or [Simple Terminal Menu](https://pypi.org/project/simple-term-menu/) or [console-menu)](https://pypi.org/project/console-menu/) to proceed the download.
